lib/countly_flutter.dart

```diff
@@ -860,7 +853,26 @@ class Countly {
     return result;
   }
 
-  static Future<String> logException(String exception,bool nonfatal, Map<String, Object> segmentation) async {
+  /// To Report Exception call logExceptionEx or logExceptionManual or logException
+  ///
+  /// Similar to [logException], but exception is [Exception] object instead of [String].
+  static Future<String> logExceptionEx(Exception exception,bool nonfatal, {StackTrace stacktrace, Map<String, Object> segmentation}) async {
+    stacktrace ??= StackTrace.current ?? StackTrace.fromString('');
+    logException("${exception.toString()}\n\n$stacktrace", nonfatal, segmentation).then((String result) {
+      return result;
+    });
+  }
+
+  /// Similar to [logException], but with optional stacktrace parameter.
+  static Future<String> logExceptionManual(String exception,bool nonfatal, {StackTrace stacktrace, Map<String, Object> segmentation}) async {
+    stacktrace ??= StackTrace.current ?? StackTrace.fromString('');
+    logException("$exception\n\n$stacktrace", nonfatal, segmentation).then((String result) {
+      return result;
+    });
+  }
+
+  /// Report handled and unhandled exception to countly.
```

Comment:
mention that this expects the stacktrace to already be added to the exception and a common usecase would be exception.toString